content/en/docs/concepts/services-networking/service.md

```diff
@@ -518,7 +518,7 @@ and `Local` to only route to ready node-local endpoints. If the traffic policy i
 endpoints, the kube-proxy does not forward any traffic for the relevant Service.
 
 {{< note >}}
-{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.26" state="beta" >}}
 If you enable the `ProxyTerminatingEndpoints`
 [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
 for the kube-proxy, the kube-proxy checks if the node
```

Comment:
```suggestion
If the `ProxyTerminatingEndpoints`
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
is enabled for the kube-proxy on a node, that node's kube-proxy uses a more
complicated algorithm to select endpoints for a Service. With the feature
enabled, the kube-proxy checks if the node
```

Reply:
Thanks, used some of your suggestion and made some addiional changes